### ISSUE TITLE
Add GPU timings to cp2k.org/performance

### ISSUE THREAD

**Issue by ltalirz** [cp2k/cp2k#1922]
If I understand correctly, all benchmarks listed in https://www.cp2k.org/performance are run without the GPU, and the last reported benchmark is from 2019.

Perhaps it would make sense to add new benchmark data at some point that also includes performance on GPUs?

**Comment by oschuett** [comment]
Yes, it would be nice to get some up-to-date GPU numbers on the wiki page.

We do keep taps on our GPU performance via some smaller benchmarks that run daily:

- https://dashboard.cp2k.org/archive/perf-cuda-volta/index.html
- https://dashboard.cp2k.org/archive/cray-xc50-daint-gpu-psmp_perf/index.html

**Comment by hfp** [comment]
I even recommend for cherry-picking workloads to also show where GPU acceleration is beneficial.

**Comment by fstein93** [comment]
On Daint, there is the 32 H2O benchmark with MP2 (and RPA) for which we can think of MP2 gradients. I don't know of what costs you have thought of. Commonly, MP2 gradient calculations are 3x as expensive as energy-only calculations. Plus, performance improvements are work in progress.

**Comment by fstein93** [comment]
We will publish an already accepted paper soon benchmarking double-hybrid gradients.

**Comment by oschuett** [comment]
Since there seems nothing left to discuss, I'm going to close this now.

Maybe with the arrival of new machines some of the performance numbers also find their way into [that wiki page](https://www.cp2k.org/performance).

**Comment by ltalirz** [comment]
> Maybe with the arrival of new machines some of the performance numbers also find their way into [that wiki page](https://www.cp2k.org/performance).

Yes, that's what the issue was opened for

**Comment by oschuett** [comment]
> Yes, that's what the issue was opened for

I'm happy to keep this issue open if you deem it useful. Personally, I see this more as a continuous maintenance task.

**Comment by ltalirz** [comment]
Cheers, I just wanted to remind the readers that the addition to the wiki would indeed be important and useful.
Feel free to keep the issue closed/open as you prefer.
